util.go

```diff
@@ -103,8 +104,19 @@ func (b Box) checkColorType(topBar, bottomBar, title string) (string, string) {
 				}
 			} else if _, ok := fgColors[str]; ok {
 				Style = fgColors[str].Sprint
-				topBar = Style(topBar)
-				bottomBar = Style(bottomBar)
+				topbars := strings.Split(topBar, "\033[0m")
+				var tmpTopBar string
+				for _, t := range topbars {
+					tmpTopBar += Style(t)
+				}
+				topBar = tmpTopBar
+
+				bottombars := strings.Split(bottomBar, "\033[0m")
+				var tmpBottomBar string
+				for _, t := range bottombars {
+					tmpBottomBar += Style(t)
+				}
+				bottomBar = tmpBottomBar
```

Comment:
I think this can be made into a function which takes `topBar` and `bottomBar` accordingly. You should also do the same for other color types/

Reply:
yes I was thinking in doing a function.

For the others color type, I can't because I don't know them (// Hi Intensity Colors ????? I have no clue)

Reply:
> yes I was thinking in doing a function.
> 
> For the others color type, I can't because I don't know them (// Hi Intensity Colors ????? I have no clue)

They are Hi Intensity Colors, this module also supports true color and 256 colors according to the terminal.